e2e/protractor/suites/info-drawer/file-folder-properties.test.ts

```diff
@@ -131,10 +131,10 @@ describe('File / Folder properties', () => {
         file1.name,
         file1.title,
         apiProps.entry.createdByUser.displayName,
-        format(apiProps.entry.createdAt, DATE_FORMAT),
+        DateFnsUtils.formatDate(apiProps.entry.createdAt, DATE_FORMAT),
```

Comment:
confirm the type of 'createdAt' & 'modifiedAt'. It should be either a number or a date object.

Reply:
Yes they are of Date(date object) types.